src/main/java/hudson/remoting/Engine.java

```diff
@@ -438,6 +439,7 @@ public void removeListener(EngineListener el) {
     }
 
     @Override
+    @SuppressFBWarnings(value = "HARD_CODE_PASSWORD", justification = "Password doesn't need to be protected.")
```

Comment:
Maybe if you put the default `password` value as a constant?

Reply:
It's smart enough to detect that attempt to deceive it.